src/sentry/api/endpoints/group_events.py

```diff
@@ -85,7 +85,11 @@ def _get_events_snuba(
             "start": start if start else default_start,
             "end": end if end else default_end,
         }
-        direct_hit_resp = get_direct_hit_response(request, query, params, "api.group-events", group)
+        referrer = f"{group.issue_category}.api.group-events"
```

Comment:
We should probably `.lower()` this as well. Might also make sense to use `name`?

```
In [4]: f"{GroupCategory.ERROR}"
Out[4]: 'GroupCategory.ERROR'

In [7]: f"{GroupCategory.ERROR.name.lower()}"
Out[7]: 'error'
```

Not sure if it also makes sense as a suffix rather than prefix